packages/blockchain-link/tests/integration/solana.ts

```diff
@@ -0,0 +1,210 @@
+import { AccountInfoParams } from '@trezor/blockchain-link-types';
+import BlockchainLink from '../../lib';
```

Comment:
Don't import from `lib` anywhere please

Reply:
I'd say that here it might be ok, this is some integration tests file. as long as it is consistent with integration test files for other backends, this is probably ok

Reply:
we will need to remove these references anyway in future, but ok :D